src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* 'az containerapp addon': support deprecation of service command in favor of addon.
```

Comment:
nit: wording sounds off, shouldn't it be "deprecation of service command and support for addon"

Reply:
yea i agree. reworded!!